src/containers/ImageEditor/ImageEditor.tsx

```diff
@@ -193,7 +193,16 @@ const ImageEditor = ({ language }: Props) => {
       <StyledImageEditorMenu>
         <FormField name="align">
           {({ field, helpers }) => (
-            <StyledToggleGroup type="single" value={field.value} onValueChange={helpers.setValue}>
+            <StyledToggleGroup
+              type="single"
+              value={field.value}
+              onValueChange={(val) => {
+                helpers.setValue(val);
+                if (val === "center") {
+                  setFieldValue("size", "full");
+                }
```

Comment:
Dette gjøres strengt tatt allerede når du submitter. Gjorde det sånn for å gjøre det likest mulig master

Reply:
Joda, men dette gjør at value oppdateres før submit og dermed så vises linja for å velge byline.

Reply:
Endra